doc/arkode/guide/source/Usage/ARKStep_c_interface/User_callable.rst

```diff
@@ -1399,6 +1399,7 @@ Specify explicit problem           :c:func:`ARKStepSetExplicit()`     ``SUNFALSE
 Specify implicit problem           :c:func:`ARKStepSetImplicit()`     ``SUNFALSE``
 Set additive RK tables             :c:func:`ARKStepSetTables()`       internal
 Specify additive RK table numbers  :c:func:`ARKStepSetTableNum()`     internal
+Specify additive RK table names    :c:func:`ARKStepSetTableName()`    internal
```

Comment:
Does this really specify the name, or does it set the table via the name provided? IMO this should read "Set additive RK tables via their name" and the one above should say "Set additive RK tables via their number".